### DISCUSSION TITLE
Region change EU to RW

### DISCUSSION THREAD

**Discussion by mrstokes60** [Mr-MIBonk/M.I.B._More-Incredible-Bash#412] [General]
Hi, I have an Aisin MIBG Discover Pro in a Golf 7 imported from Japan. My aim is to install ROW maps for use in New Zealand. 
I have used MIB 3.7.1 'eu conversion' because I have seen a successful conversion using EU firmware. I'm asking for advice before I attempt any more. This is my plan.
 GEM-> MIB -> Navigation -> check coding -> change Navigation country to Australia. Change SDS region flag to Australia -> Press to save changes.
              Advanced settings -> format_nav_db -> Format Navigation DB.
Do I reboot to save changes?

Should I change this before or after map install?. Navigation -> nav_activation -> Enable Australia navigation.

Exit GEM. Update map data.

I have uploaded 2 backups. 1st  before any changes and 2nd after eu conversion.
I would appreciate any help with this.  Thanks  Pete

![5F label_Aisin](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/157345519/292e90c6-e1e5-4329-bb9f-72da9de9f611)
![MHIG_EU](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/157345519/8737389e-5940-4992-be37-1724e49faa86)
![MHIG_JP_](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/assets/157345519/7b8bd182-e5ca-47a4-a4a4-6872b50bcf72)

**Comment by krezo888** [comment]
Hi there, I'm in the same boat where I also have a Japanese import golf mk7 (MHIG unit) and I'm trying to convert the region and firmware to install ROW maps for NZ.

Did you end up installing the ROW maps for NZ fine with the EU firmware following the steps you took?
Also, why did you use EU region and firmware instead of ROW (MHIG_RW_VW_S1534), would the ROW maps work fine on EU firmware (MHIG_EU_VW_K1550)? 

Cheers!

**Reply by aledepaoligampel** [reply]
I have the same problem, the EU maps on MHIG_EU works fine, but RoW maps not. I have succesfull installed the FEC 021D00EE and its legal in the red menu. But I got the Data are invalid. I tried everything. Im from Argentina BTW. Any help is welcome to us.

**Reply by krezo888** [reply]
The EU maps work fine for me too on MHIG_EU train, fecs for RoW are also applied and valid so I think that's not an issue. Also I tried older RoW maps, installed fine on ROW 'variant info' but still didn't work, that's with code nav and SDS set to Australia.

I think RoW maps are not compatible with MHIG_EU train, we must somehow convert to MHIG_RW train.

**Reply by pixeleyesltd** [reply]
I managed to get NZ / New Zealand maps on a JP / Japan unit in case you are still needing the details. This is applicable for firmware version MHI2_ER_VWG11_K3342_1_AIO_MU1427_20230318, maybe others. My problem: after EU conversion and firmware update using the AIO firmware method, ROW maps would not load. The M.I.B. add FEC step did not function for me, perhaps due to the AIO procedure. This prevented ROW maps from being installed, but there were other steps involved too as the regions were incorrect. See below steps for how I resolved it.

- Generated FEC SWAP code via https://vwcoding.ru/en/utils/fec/
  - FEC "093d0008" which is ROW Maps indefinitely. See FEC decoder for details: https://mib-helper.com/fec-decoder/?fec=093d0008
  - I used browser web inspector to change one of the HTML checkbox values to the FEC above, which when checked and combined with your VIN and VCRN generated a valid SWAP code for me.
- Loaded SWAP code via OBDEleven adaption channel, and activated with basic setting: https://www.youtube.com/watch?v=rgAVFoQQ6v4. It then shows up in the Installed FEC list.
- Using M.I.B.: 
  - Cleaned the Nav DB database
  - Set Variant region to RoW
  - Set navigation region and SDS region to Australia
  - Reboot
- Load the latest 2024/2025 RoW maps onto appropriately sized FAT32 formatted SD card, downloaded from VW website. If using macOS, I deleted all ._ files in the SD card folder structure after the copy.
- Initiate software update via long press menu > SWUpdate. All NavDB, eggnog etc in the list should say Y, and no errors should occur. The update was successful, and on reboot navigation worked without issue.

Hope that helps someone! NZ maps do work on EU firmware with some coaxing.

**Reply by kurlykiwi** [reply]
@pixeleyesltd So you used the AIO FW version from mibsolution? I have an A4 and it has MHI2_JP_AUG22_PO623 MU 0623 software and im trying to work out which FW to download. Thanks

**Reply by kurlykiwi** [reply]
@pixeleyesltd Managed to get the FW sorted and the infinite maps update installed. Despite cleaning the Nav DB, setting variant to ROW and region and SDS to Australia, I still cant get the maps to be recognized? I downloaded the ROW maps (P393_N60S5MIBH3_ROW) from mibsolution.one but they wont install? Any advice?

**Comment by kurlykiwi** [comment]
OK, so latest update. As above i have managed to update MMI FW from MHI2_JP_AUG22_PO623 MU 0623 to MHI2_ER_AUG22_K3346_MU1438. I then patched to get unlimited maps etc using MHI2_ER_AUG22_K3346_MU1438_PATCH and used MIB to set the region to AU and Nav DB variant to RoW etc. I then downloaded P393_N60S5MIBH3_ROW maps and installed these successfully (through the REM menu). However, after this completing successfully, i get a "The Navigation data are invalid" error when trying to start navigation? Can anyone help? Thanks

**Reply by SLOOGOVS** [reply]
@kurlykiwi - Please can you help us understand how you converted from EU to ROW? There is no information anywhere on how to do this exact conversion from what I've read on Github or outside of it... My guess is that I do the following before installing firmware, but a lot of us would greatly appreciate it if you can spell out exactly what you did!

Can I change my firmware from MHIG_EU_AU_K1555 to MHIG_RW_AU_K1556, re-patch in M.I.B, and install the RoW maps by doing the following?:

- Change the region to ROW in the variantinfo menu
- Change my navigation region to the Australian navigation region
- Enable user-defined mode and ignore region and variant in the SWDL menu
- Install MHIG_RW_AU_K1556 via the red engineering menu

Am I missing anything? what other settings or flags the 'convert to EU' function change, and can this be done manually so that I can install RoW firmware?
